wiki/Beatmap_Spotlights/zh.md

```diff
@@ -174,14 +172,36 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
+
+2020 年 8 月 ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) 退出了该项目，在 2020 年 11 月底 ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) 也随 Kurokami 辞职。于是 ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) 和 ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) 一起接管了该项目。
 
 在第 4 赛季（2021 春）后，项目进入停顿期。直至 2021 年 9 月，出现了复兴的倾向，::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) 卸任，而 ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) 加入管理团队，于 2022 年 2 月，在人员重组后，项目恢复了运作。
+
+## 参见
+
+[^charts-09-oct]: [Forum thread by peppy (2009-10-25) "osu! Public Release b1077"](https://osu.ppy.sh/community/forums/topics/19115)
+[^charts-09-nov]: [(Restricted) Forum thread by Cyclone (2009-11-03) "'Monthly Chart' discussion November edition"](https://osu.ppy.sh/community/forums/topics/19560)
+[^charts-10-jan]: [Forum thread by Cyclone (2009-12-10) "Best Beatmaps of 2009!"](https://osu.ppy.sh/community/forums/topics/21059)
+
+[^charts-11-per-mode]: [Forum thread by Cyclone (2011-08-24) "Apply to help create ranking charts! (osu!, Taiko, or CtB)"](https://osu.ppy.sh/community/forums/topics/60660)
+[^charts-manager-sg]: [Forum thread by Cyclone (2012-05-20) "May 2012 Ranking Chart, and new Chart Manager"](https://osu.ppy.sh/community/forums/topics/84573)
+[^charts-manager-dxs]: [Forum thread by DeathxShinigami (2013-04-14) "April 2013 Ranking Chart"](https://osu.ppy.sh/community/forums/topics/127847)
+[^charts-manager-makar]: [Forum thread by Makar (2013-05-19) "2013 NewCAT Applications"](https://osu.ppy.sh/community/forums/topics/133248)
+[^charts-manager-loctav]: [(Restricted) Forum post by Loctav (2013-11-21) in "Regarding the charts"](https://osu.ppy.sh/community/forums/posts/2697871)
+[^charts-in-osu-14-jan]: [(Restricted) Forum post by peppy (2014-01-24) in "So peppy's planning something chart related"](https://osu.ppy.sh/community/forums/posts/2824323)
+[^charts-cat-recruitment-14-jun]: [(Restricted) Forum thread by Loctav (2014-06-16) "\[IMPORTANT\] Recruiting NewCATs"](https://osu.ppy.sh/community/forums/topics/218032)
+[^charts-curated-15-mar]: [News post by Loctav (2015-03-18) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
+[^charts-reverted-16-sep]: [News post by OnosakiHito (2016-09-17) "July 2016 Ranking Charts - Changes"](https://osu.ppy.sh/home/news/2016-09-17-july-2016-ranking-charts-changes)
+
+[^charts-renamed-into-spotlights]: [News post by OnosakiHito (2017-03-18) "Introducing to you: Spotlights"](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
+[^spotlights-seasonal]: [News post by Kurokami (2018-11-01) "Seasonal Spotlights: Summer 2018"](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)
+[^spotlights-reworked-20-june]: [Forum thread by Loctav (2020-07-06) "Beatmap Spotlights (Summer 2020) - Discussion Thread"](https://osu.ppy.sh/community/forums/topics/1101170)
```

Comment:
references need to be translated, too (except for the thread and article titles). example: https://github.com/ppy/osu-wiki/pull/8265